client/app/states/designer/rules/rules.state.js

```diff
@@ -0,0 +1,214 @@
+(function() {
+  'use strict';
+
+  angular.module('app.states')
+    .run(appRun);
+
+  /** @ngInject */
+  function appRun(routerHelper) {
+    routerHelper.configureStates(getStates());
+  }
+
+  function getStates() {
+    return {
+      'designer.rules': {
+        parent: 'application',
+        url: '/designer/rules',
+        templateUrl: 'app/states/designer/rules/rules.html',
+        controller: RulesController,
+        controllerAs: 'vm',
+        title: N_('Rules'),
+        resolve: {
+          designerRules: resolveRules,
+          fields: resolveFields,
+          profiles: resolveProfiles
+        }
+      }
+    };
+  }
+
+  /** @ngInject */
+  function resolveRules(CollectionsApi) {
+    var options = {
+      expand: 'resources'
+    };
+
+    return CollectionsApi.query('arbitration_rules', options);
```

Comment:
One note, wondering if these CollectionApi calls should be separated into a service rather than being in the rules.state file, which may get rather large as more functionality is added to the Rules List view.  See ../client/app/services/blueprints-state.service.js for an example.